compiler.rst

```diff
@@ -7,23 +7,14 @@ Design of CPython's Compiler
 Abstract
 --------
 
-Historically (through 2.4), compilation from source code to bytecode
-involved two steps:
+In CPython, the compilation from source code to bytecode involves three steps:
```

Comment:
Thanks for the review @willingc (nice tool, too!). I updated the text to use 'several'.